roles/provision-keycloak-apb/files/keycloak-metrics-spi-version.txt

```diff
@@ -0,0 +1 @@
+1.0.1-SNAPSHOT
```

Comment:
Would it be useful here to replace `-SNAPSHOT` with `-<shortcommit>` or something?

Reply:
not really sure, but this is the version in the metrics.

Reply:
It's actually probably fine, since Ali added a plugin that updates the MANIFEST.MF in the jar to show what commit it was built from (a plugin that we currently remove downstream).